### PR TITLE
Stats: removed the logo form dashboard widget title

### LINKED ISSUE #22669: Admin Performance: Remove duplicate Jetpack SVGs loading in HTML
On `wp-admin/index.php`, the HTML included 5 copies of the same 1.6kb SVG embedded in the code:

image

I recommend keeping the Jetpack logo in its own SVG file, so the browser can cache it.

Internal reference: p1HpG7-evU-p2

### PR DESCRIPTION
**Pull request by bindlegirl**
The Jetpack logo that was added to the dashboard stats widget gets loaded several times on the same page, and was causing performance and accessibility issues. The logo gets attached to the move up & down, and toggle widget buttons. 
For that reason, screen readers read buttons as "Move Stats by box up...". 

This issue is also related to the broken Screen options issues mentioned  here: https://github.com/Automattic/jetpack/pull/22812

Fixes #22669

#### Changes proposed in this Pull Request:

* Removed the logo from the dashboard widget title. The title is now plain text "Stats by Jetpack".
* Created a new widget footer section and added the logo there to keep the branding consistency. 
* Added Configure Stats and Learn More links to the new footer section.
* Applied styles from the Jetpack dashboard to the blocked numbers in the widget for better style consistency.

How the widget looked before:

stats_widget_before

How it looks now:

stats_widget_after

#### Jetpack product discussion

N/A

#### Does this pull request change what data or activity we track or use?

No

#### Testing instructions:

* Go to WP Admin and see how the widget looks with different screen widths.
* Try disabling, Stats, Protect, and Akismet.
* Check the Screen options.

### CODE REVIEW

File: projects/plugins/jetpack/changelog/fix-remove-duplicate-svg-logo [hunk 1]
```diff
@@ -0,0 +1,4 @@
+Significance: patch
+Type: other
+
+Removed the SVG logo from the dashboard stats widget. Created a new footer section for the widget witlinks
```
Comment (samiff) [inline projects/plugins/jetpack/changelog/fix-remove-duplicate-svg-logo:4]:
```suggestion
Stats: improve accessibility and performance for the admin dashboard widget.
```

File: projects/plugins/jetpack/class-jetpack-stats-dashboard-widget.php [hunk 2]
```diff
@@ -89,69 +75,94 @@ public static function dashboard_widget() {
...
+					endif;
 				?>
-							" class="button button-jetpack">
-					<?php esc_html_e( 'Activate Anti-spam', 'jetpack' ); ?>
-				</a>
-			<?php else : ?>
-				<p><a href="<?php echo esc_url( 'https://akismet.com/?utm_source=jetpack&utm_medium=link&utm_campaign=Jetpack%20Dashboard%20Widget%20Footer%20Link' ); ?>"><?php esc_html_e( 'Anti-spam can help to keep your blog safe from spam!', 'jetpack' ); ?></a></p>
-			<?php endif; ?>
-		</div>
+			|
```
Comment (samiff) [inline projects/plugins/jetpack/class-jetpack-stats-dashboard-widget.php:159]: The `|` is outside the `if` so will display even if a user can't `jetpack_manage_modules`.

**Review by samiff — CHANGES_REQUESTED** [review]
Looking good and I saw only one Jetpack logo being loaded 👍

**Review by keoshi — APPROVED** [review]
Design-wise LGTM!

File: projects/plugins/jetpack/class-jetpack-stats-dashboard-widget.php (hunk 2 above)
Comment (bindlegirl) [reply projects/plugins/jetpack/class-jetpack-stats-dashboard-widget.php]: Ah, good catch. Thanks! Fixed in 9fbd77ab1a007e58bf12c14e062ab8955ede4346

**Pull request merged**